examples/HelloSO3.cpp

```diff
@@ -0,0 +1,40 @@
+#include "sophus/geometry.hpp"
+#include <iostream>
+
+int main() {
```

Comment:
I don't think this file is formatted correctly. Make sure to run "run_format.sh" and resubmit.